lib/config/options.js

```diff
@@ -217,6 +217,11 @@ module.exports = {
                 height: parseInt(size[1])
             };
         }
+    },
+
+    referenceImageAbsence: {
+        default: 'error',
+        validate: 'string'
```

Comment:
I would prefer more strict validation: does not allow any value except `error` or `warning`. It can be done in `map` method.